conf.yaml

```diff
@@ -904,8 +904,8 @@ contents:
             current:    7.0
             branches:   [ master, 7.0, 6.7, 6.6, 6.5, 6.4, 6.3, 6.2, 6.1, 6.0 ]
             chunk:      1
-            tags:       APM Server/Reference
-            subject:    APM
+            tags:       APM/Elastic APM/APM Overview/APM Getting Started/Install APM/Run APM
+            subject:    Elastic APM Overview
```

Comment:
I poked around a bit just now and it seems like the "subject" is used in the "Product name" filter on our search results, so you might want to consider whether this particular value is appropriate here.

Reply:
Ahh crap. That explains why every subject is so simple. Sounds like a better solution might be having a different yml key map to DC.subject. But that's not the easiest of fixes. OK, I'll play around a bit more. Might have to cancel this idea for now. Thanks for your expertise.